gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -13,26 +13,174 @@
 # limitations under the License.
 
 """beam combiner function for merging VCF file headers."""
-
+from collections import OrderedDict
 import apache_beam as beam
-from gcp_variant_transforms.beam_io import vcf_header_io
 
-__all__ = ['MergeHeaders']
+from gcp_variant_transforms.beam_io.vcf_header_io import VcfHeader
```

Comment:
nit: google style requires us to import modules and not classes. This is not enforced by pylint and is not actually a pattern in Beam, but we have generally done this in most of our code (other than beam_io/)

Reply:
Done